pages/api/graphql.ts

```diff
@@ -31,7 +50,6 @@ handler
       resave: false,
       saveUninitialized: false,
       cookie: {
-        domain: process.env.DB_HOST,
```

Comment:
Why is domain being removed?

Reply:
isn't the default `domain` the some host?

Reply:
When I was testing session, it didn't have any `domain` in the cookie if I don't put that.